core/tracepoint/new_spec.rb

```diff
@@ -0,0 +1,9 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+ruby_version_is '2.4' do
+  describe 'TracePoint.new' do
```

Comment:
Needs additional specs for being called with a `String`, with no argument, with multiple arguments.

Reply:
okay, I have updated the PR.